linux_files/setup

```diff
@@ -722,6 +722,9 @@ fi
 function dockerinstall_build_relay {
 
     #Build the relay
+
```

Comment:
The problem is wslupath has hardcoded the root in /mnt/c. If the user or wlinux-setup dockerinstall change the root, wslupath will return the wrong path.